### PR TITLE
docs: added changes log for release version 2.14.6

### PR DESCRIPTION
**Pull request by AmraniCh**
As the title says, adding changes logs for version `2.14.6`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# 2.14.6
+
+* Added support of PHP version `8.2` for GitHub workflow tests ([#740][i740]).
+* Upgraded prismJS to version `1.29.0` due to security issue ([#741][i741]).
+
+[i740]: https://github.com/filp/whoops/pull/740
+[i741]: https://github.com/filp/whoops/pull/741
```
Comment (denis-sokolov) [inline CHANGELOG.md:7]: I’ll remove the note of our internal work. It is not relevant to the consumers of the package, who read the changelog. Let me know if you mind it!

```suggestion
* Upgraded prismJS to version `1.29.0` due to security issue ([#741][i741]).

[i741]: https://github.com/filp/whoops/pull/741
```

**Review by denis-sokolov — COMMENTED** [review]
Thank you!

File: CHANGELOG.md (hunk 1 above)
Comment (AmraniCh) [reply CHANGELOG.md:7]: You're right it is relevant more to developers and maintainers, but in my opinion, It is a good practice to log everything either the internal relevant changes, I suggest to have like a standard for our change log file, for ex we can prefix the internal logs with `[Internal]: log_description`, so our two change logs will look like:

```markdown
* Added support of PHP version `8.2` for GitHub workflow tests ([#740][i740]).
* [Internal]: Upgraded prismJS to version `1.29.0` due to security issue ([#741][i741]).
```

Of course, all the internal changes should be added last and with each other, what do you think ?

Comment (denis-sokolov) [reply CHANGELOG.md:7]: I’d like to keep the main changelog focused on the information our consumers care about. If you want to maintain a separate internal changelog, I will merge that. Although evaluate whether the [list of commits](https://github.com/filp/whoops/commits/master) doesn’t solve that problem already.

**Comment by denis-sokolov** [comment]
I’ve added your commit partially to the changelog (9c68d0d8b39). Thank you for correcting the oversight with the missed changelog entry!

File: CHANGELOG.md (hunk 1 above)
Comment (AmraniCh) [reply CHANGELOG.md:7]: Okay, I will create a separate change log file (`CHANGELOG_INTERNAL.md`) for our internal relevant changes, I will maintain this file and watch the overall changes in Whoops and detect undocumented internal work changes.

Comment (AmraniCh) [reply CHANGELOG.md:7]: What do you think ? Or for things to be more organized we can wait until the next major update, then we start to log our internal changes.

Comment (denis-sokolov) [reply CHANGELOG.md:7]: I don’t think it‘s particularly valuable, but I also wouldn’t stop you if you think it is!

**Pull request closed without merging**